app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -109,9 +109,6 @@
     --tblr-btn-padding-y: 4px;
   }
 
-  // variable that did not exists in tabler but used by selectize.
-  --bs-border-color-translucent: var(--tblr-border-color);
```

Comment:
used to set border color for dropdown, but became unnecessary

Reply:
or just
```css
--bs-border-color-translucent: var(--tblr-border-color-translucent);
```